tests/unit/nginx-vhosts.bats

```diff
@@ -109,21 +109,11 @@ assert_error_log() {
   assert_output "0"
 }
 
-@test "(nginx-vhosts) nginx:build-config (with SSL CN mismatch)" {
-  setup_test_tls
-  deploy_app
-  assert_ssl_domain "node-js-app.dokku.me"
-}
-
-@test "(nginx-vhosts) nginx:build-config (with SSL CN mismatch & custom nginx template)" {
-  setup_test_tls
-  custom_ssl_nginx_template
-  deploy_app
-  assert_ssl_domain "node-js-app.dokku.me"
-}
-
```

Comment:
Removing functionality is still a BC break. We're pretty close to a 0.4.0, so I think we can wait till then, yes? May as well rip off all the Band-Aids then :)


(Our next release can probably be 0.4.0 unless herokuish has any bugs.)